frontend/public/components/dashboard/with-dashboard-resources.tsx

```diff
@@ -104,7 +92,7 @@ export const withDashboardResources = <P extends DashboardItemProps>(
           urlResultChanged ||
           queryResultChanged ||
           k8sResourcesChanged ||
-          (this.watchingAlerts && alertsResultChanged) ||
+          alertsResultChanged ||
```

Comment:
this would cause unnecessary card rerendering even if it does not need to watch alerts.
I'd like to get rid of this HOC in the future - I'd prefer to connect the specific cards to redux/notificationAlerts instead

Reply:
Good point @rawagner.  I could restore the `watchAlerts` functions/vars, minus the polling, in order to keep the `watchingAlerts` functionality in place to avoid unnecessary card rerendering (although in my limited experience I don't see alertsResultChanges that often), or do you feel the best solution is to connect the specific cards to redux/notificationAlerts in the future?

Reply:
restoring the function would be fine

Reply:
Hi @rawagner, is the :+1: for restoring watchAlerts or waiting to connect specific cards to redux/notificationAlerts in the future?

Reply:
we can go with restoring the function for now. I will start getting rid of this HOC once #3443 is merged

Reply:
Hi @rawagner, restored `watchAlerts` functionality.  PTAL